File: Udemy/Algoritmos e Logica de Programacao/Java/ExerciciosJavaVetores/src/application/ProblemaSomaVetor.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaSomaVetor {

	public static void main(String[] args) {
		/*
		 * Problema "soma_vetor"
Faça um programa que leia N números reais e armazene-os em um vetor. Em seguida:
- Imprimir todos os elementos do vetor
- Mostrar na tela a soma e a média dos elementos do vetor
		 * */
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Quantos numeros voce vai digitar? ");
		int n =sc.nextInt();
		double[] vet = new double[n];
		double soma =0.0;
		double media =0.0;
		for(int i=0;i<n;i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
			soma = soma + vet[i];
		}
		
		media = (double)soma / n;
		System.out.print("VALORES = ");
		for(int i=0;i<n;i++) {
			System.out.print(String.format("%.2f ", vet[i]));
		}
		System.out.println();
		
		System.out.println("SOMA = "+String.format("%.2f", soma));
		System.out.println("MEDIA = "+String.format("%.2f", media));
		
		sc.close();
	}

}

File: Algoritmos e Logica de Programacao/Python/Vetores/ProblemaSomaVetor.py
'''Problema "soma_vetor"
Faça um programa que leia N números reais e armazene-os em um vetor. Em seguida:
- Imprimir todos os elementos do vetor
- Mostrar na tela a soma e a média dos elementos do vetor'''

n = int(input("Quantos numeros voce vai digitar? "))
soma : float; media: float;
soma = 0.0
media = 0.0
vet : [float] = [0 for x in range(n)]

for i in range(0,n):
        vet[i] = float(input("Digite um numero: "))
        soma = soma + vet[i]


media = soma / n
print("VALORES = ", end="")
for i in range(0,n):
        print(f"{vet[i]:.1f} ", end="")

print()
print(f"SOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")